routes/admin.js

```diff
@@ -1,13 +1,14 @@
 const express = require('express');
 
 const validationController = require('../controllers/validationController')
-const indexController = require('../controllers/playersIndexController');
+const adminController = require('../controllers/adminController');
 
 const router = express.Router();
 
 // Get all players info
 // /admin/players => GET
 // validate a password to get players infos
-router.get('/players/', validationController.validateAdminPassword, indexController.getIndex);
+router.get('/players/', validationController.validateAdminPassword, adminController.getIndex);
+router.get('/delete-players/', adminController.postDeletePlayer);
```

Comment:
This should be `router.delete('/players')`. 

The REST API is designed this way. We will have one named endpoint with different http method. GET/POST/PUT/DELETE and more. So in one named endpoint, it can do all CRUD (create, read, update, delete).